fishtest/fishtest/rundb.py

```diff
@@ -647,3 +653,20 @@ def update_spsa(self, worker, run, spsa_results):
 
     if grow_summary:
       spsa['param_history'].append(summary)
+
+@jit(nopython=True)
+def jit_spsa_param_clip_round(theta, pmin, pmax, increment, clipping):
+  value = 0.0
+
+  if clipping == 'old':
+    value = min(max(theta + increment, pmin), pmax)
+  else: #clipping == 'careful':
+    inc = min(abs(increment), abs(theta - pmin) / 2, abs(theta - pmax) / 2)
+    if inc > 0:
+        inc_sgn = 0 if increment == 0 else increment / abs(increment)
```

Comment:
if `increment == 0` then `inc == 0` so the ternary operator can be dropped => `value = theta + inc * increment / abs(increment)`

Reply:
Yes, you are correct! But most (all?) tests are run with `clipping == 'old'` so no speedup in general case.

Will you create a new PR for `spsa_param_clip_round`?
Note that the rounding code can also be removed (line 538) from that function

Reply:
@tomtor for the general case we can try replacing `min(max())` w/ `sorted` and bench for a speedup.
I'm not the best person to submit a new PR:
- I don't know the whole SPSA code and how to bench the speedup for that code
- I don't know how to submit a new SPSA test (always reschedule a recent one)

Reply:
OK, but I will not introduce the `sorted` because the speed difference is small and creating the list takes time.